src/screens/Dashboard/Dashboard.js

```diff
@@ -332,63 +241,50 @@ export default function HrDashboard(props) {
     });
   };
 
+  const loadData = () => {
```

Comment:
Use try catch on all api calling.
check for res.data.status as well

Reply:
it will crash and give promise error while using try and catch